lib/services/identity/controllers/identity-config-controller.js

```diff
@@ -17,6 +19,24 @@ class IdentityConfigController {
 		this.bus
 			.query({role: 'identity', cmd: 'config'}, {channel, platform, user})
 			.then(resource => {
+				if (user) {
+					return resource;
+				} else { // eslint-disable-line no-else-return
```

Comment:
It complains with the no return OR !user negation. Either way its going to complain so I went with readable positive case.